data/panel.css

```diff
@@ -229,7 +230,6 @@ input[type=range]::-moz-focus-outer {
 
 .video-progress {
     background: #ffffffaa;
-    border-radius: 20px;
```

Comment:
@6a68 went back and forth about different ways to get the `[value]` to be the same shape as the `.video-progress` bar at < 2 seconds. I looked at svg masks, the background-clip css property, and few other things, but eventually it seemed to be a lot of work for such a small thing. I also wasn't sure how a mask would scale as the player gets resized. So we decided that the way to most consistently get the [value] bar to be the right shape was just to get rid of the border-radius.